web/client/translations/data.en-US.json

```
@@ -1125,12 +1125,13 @@
             "changePwd": "Change Password",
             "newPwd": "New Password",
             "retypePwd": "Retype Password",
-            "passwordMinlenght": "Your password must be at least {minSize} character",
+            "passwordMinlenght": "Your password must be at least {data} character",
             "passwordCheckFail": "Passwords do not match!",
             "passwordInvalid": "Invalid password",
             "username": "Username",
             "password": "Password",
-            "passwordMessage": "Password must contain at least 6 characters",
+            "passwordMessage": "Password must contain at least 6 characters. Allowed characters !,@,#,$,%,&,*,_",
```

```suggestion
            "passwordMessage": "Password must contain at least 6 characters. Valid values for passwords include numerals, lower or upper-case letters and the following special characters: ! @ # $ % & * _.
```
and other translation files

---

verify this works, because in the screenshot i see it truncated

---

@MV88 I have verified, it doesn't look truncated.